Update resource icons

avatus opened the pull request:
This will update the kubeserver, server, application, and desktop icons to the new, shadowless version
![Screenshot 2024-08-28 at 12 10 42 PM](https://github.com/user-attachments/assets/ff634f80-46f2-4245-a5eb-4787d187f623)
![Screenshot 2024-08-28 at 12 10 53 PM](https://github.com/user-attachments/assets/5680fcb0-5501-452b-8585-549274248662)
![Screenshot 2024-08-28 at 12 11 04 PM](https://github.com/user-attachments/assets/58155ec5-ee9a-4775-baaa-2d6740fd36e3)


Note: the resources cards will still use the Kube logo to better distinguish from servers (per UX team). I've updated the kubeserver logo in this PR but I don't believe it is actually used anywhere in the app.

I browsed every resource type in my cluster (which is at least one of every type in the unified cache) and the icons all are shadowless and seem to have matching styles now. 

Happy to add on any other icon changes we might need to this PR if anyone finds anything else lingering

changelog: Updated the icons for server, application, and desktop resources

zmb3 commented on web/packages/design/src/ResourceIcon/assets/windows-light.svg:
`windows-dark.svg` is still using the "trapezoid" logo. Should we be consistent?

avatus replied:
Yeah. the resource icons don't actually use the windows dark anywhere but I will update

avatus committed:
Update resource icons

This update the kubeserver, server, application, and desktop icons to
the new, shadowless version

avatus commented:
Note: the resources cards will still use the Kube logo to better distinguish from servers (per UX team). I've updated the kubeserver logo in this PR but I don't believe it is actually used anywhere in the app.
